spec/views/helpers/page_images_helper_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe PageImagesHelper, type: :helper do
+  before do
+    CloudConfig.class_eval { remove_const(:SERVICE) }
+    CloudConfig::SERVICE = SlideHub::Cloud::Engine::AWS
+    CloudHelpers.switch_to_aws
+  end
+
+  describe 'list_tag' do
+    it 'returns tag that include actual images' do
+      slide = FactoryBot.create(:slide)
+      tag = helper.slide_page_image_list_tag(slide)
+      expect(tag).to match(%r{<li><img class="lazy" src="http://test.host/assets/spacer-(.*).png" data-original="https://my-image-bucket.s3-ap-northeast-1.amazonaws.com/(.*)/slide-1.jpg" /></li>})
```

Comment:
__[RuboCop]__ Metrics/LineLength: Line is too long. [196/160]

<sub>posted by Sider</sub>